app/controllers/api/v1/websites_controller.rb

```diff
@@ -1,9 +1,16 @@
 class Api::V1::WebsitesController < Api::V1::ApiController
+  skip_before_action :authenticate_member!, except: [:index]
 
   def index
     @websites = current_member.websites
   end
 
   def create
+    @website = Website.create(domain: website_params[:domain], title: website_params[:title], member_id: current_member.id)
   end
+
+  private
+		def website_params
+			params.require(:website).permit(:domain, :title, :id, :app_id, :app_key)
```

Comment:
nu dam voie aici sa se updateze `app_id` si `app_key` stergele. App_id si app_key or sa fie creeate cand creeam un site in model.